### ISSUE TITLE
[Node] Add Playwright into test suite

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#269]
For the Node tool, there has been a request for Playwright support:
https://ibm-studios.slack.com/archives/C036P1CTN/p1603898867297500

**Comment by darekkay** [comment]
Hello @tombrunet , this was my request. I've had a look and I've made the accessibility-checker compatible with Playwright locally. It requires only a small change regarding the `evaluate` function. I will provide a PR to address it.

**Comment by tombrunet** [comment]
Reopening - want to look at expanding the test suite to cover

**Comment by tombrunet** [comment]
Now included in PR tests
